src/common/DeveloperConfig.ts

```diff
@@ -23,12 +24,20 @@ import { SvgProps } from 'react-native-svg';
  * @param additionalNavigationTab Allows for configuring a custom additional material
  * bottom navigation tab (in addition to Home, Notifications, and Settings.
  */
+export interface RouteColor {
+  route: string;
+  color: (theme: Theme) => string;
```

Comment:
Instead of individual colors, I think we would get more mileage out of being able to specify a viewStyle or textStyle per route

Reply:
Oh, nice idea.  I like.